### PR TITLE
Ui 

### PR DESCRIPTION
**Pull request by japneetbhatia**

### CODE REVIEW

File: src/components/FilterIcon.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { Grid } from '@mui/material';
+import DirectionsCarFilledIcon from '@mui/icons-material/DirectionsCarFilled';
+import DirectionsSubwayIcon from '@mui/icons-material/DirectionsSubway';
+import './FilterIcon.css';
+
+const style = {
+  borderRadius: '20px', marginLeft: '45px', borderColor: '#1976e9',
+};
+
+const FilterIcon = () => (
```
Comment (hariharanweb) [inline src/components/FilterIcon.js:13]: is it icon? or filter section

File: src/screens/InitScreen.js [hunk 2]
```diff
@@ -62,8 +63,9 @@ const InitScreen = () => {
   };
   return (
     <>
-      <Header title="Payments" onBackClick={gotoHome} />
+      <Header onBackClick={gotoHome} />
```
Comment (hariharanweb) [inline src/screens/InitScreen.js:66]: think about some encapsulation in next MR

**Review by hariharanweb — CHANGES_REQUESTED** [review]

File: src/components/FilterIcon.js (hunk 1 above)
Comment (japneetbhatia) [reply src/components/FilterIcon.js]: Updated

File: src/screens/InitScreen.js (hunk 2 above)
Comment (japneetbhatia) [reply src/screens/InitScreen.js:66]: okay

**Pull request merged**